faq.md

```diff
@@ -233,11 +233,9 @@ The primary criteria we used to pick a programming language were the following:
   performance and delivers predictable performance, without periodic
   pauses that would cause dropped frames.
 
-* _Fast allocation_. The Flutter framework uses a functional-reactive style
-  programming model, whose performance depends heavily on the underlying memory
-  allocator efficiently handling small, short-lived allocations. The
-  functional-reactive style was developed in languages with this property and
-  does not work efficiently in languages that lack this facility.
+* _Fast allocation_. The Flutter framework uses a functional-style flow
+  that depends heavily on the underlying memory
+  allocator efficiently handling small, short-lived allocations.
```

Comment:
Well if we just leave it the way it was then I don't think we need to particularly change anything. :-)

@abarth is the one who originally wrote this sentence so he might have thoughts on how to rephrase it if we want to remove mention of "functional-reactive" but keep it accurate.